cpu/stm32/stmclk/stmclk_l0l1.c

```diff
@@ -254,5 +407,31 @@ void stmclk_init_sysclk(void)
         stmclk_disable_hsi();
     }
 
+    /* Enable the LSE if used for MCO */
+    if (IS_ACTIVE(CLOCK_ENABLE_LSE)) {
+        stmclk_dbp_unlock();
+        RCC->CSR |= RCC_CSR_LSEON;
+        while (!(RCC->CSR & RCC_CSR_LSERDY)) {}
+        stmclk_dbp_lock();
+    }
+
+    /* Enable the LSI if used for MCO */
+    if (IS_ACTIVE(CLOCK_ENABLE_LSI)) {
+        RCC->CSR |= RCC_CSR_LSION;
+        while (!(RCC->CSR & RCC_CSR_LSIRDY)) {}
+    }
```

Comment:
Is there no case for enabling `LSI` or `LSE` not for MSO, can't it be used to clock RTT for example?

Reply:
Comments added like in #15064